dataprofiler/data_readers/data_utils.py

```diff
@@ -4,13 +4,16 @@
 from io import open, StringIO, BytesIO, TextIOWrapper
 from collections import OrderedDict
 import dateutil
+import logging
 
 import pandas as pd
 import pyarrow.parquet as pq
 from chardet.universaldetector import UniversalDetector
 
 from .filepath_or_buffer import FileOrBufferHandler
 
+logger = logging.getLogger('DataProfiler.data_utils')
```

Comment:
needs 2 spaces from import

Reply:
Also, if we always do `DataProfiler.{}` there might be collisions, no? Should we instead do something like 
`logger = logging.getLogger(__FILE__)` or in this case `logger = logging.getLogger('dataprofiler.data_readers.data_utils')`? where we simulate based on module?

Reply:
Yeah thats a good point, I'll add.